zm/zm_convr.F90

```diff
@@ -1907,7 +1606,8 @@ SUBROUTINE ientropy (rcall,icol,s,p,qt,T,qst,Tfg,cpliq,cpwv,rh2o,errmsg,errflg)
 ! for T and saturated vapor mixing ratio
 !
 
-  use phys_grid, only: get_rlon_p, get_rlat_p
+! CACNOTE - Remove this when pass in lat/lon or pass out lchnk,icol
+!  use phys_grid, only: get_rlon_p, get_rlat_p
```

Comment:
I am not sure if passing out `lchnk,icol` will work because there is no `lchnk` in CAM-SIMA.  Just an FYI to future you.

Reply:
Noted this in my comment in the code